help.html

```diff
@@ -26,23 +26,23 @@
     <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
     <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
     <script src="static/js/ie-emulation-modes-warning.js"></script>
-
+    <script src="urlparser.js"></script>
     <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
     <!--[if lt IE 9]>
       <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
 :q      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
     <script>
         $(document).ready(function() {
-            window.location.search.substring(1).split("&").map(function(param) {
-                return param.split("=");
-            }).filter(function(pair) {
-                return pair[1] == "true";
-            }).forEach(function(pair) {
-                var enabledFlag = decodeURIComponent(pair[0]);
-                var hiddenFeatureClass = 'hide-' + enabledFlag;
-                $("." + hiddenFeatureClass).removeClass(hiddenFeatureClass);
-            });
+            let paramsToCheck = ["snippets","blocks"];
```

Comment:
`let` is an ES6 keyword, we should stick to ES5 to maintain compatibility with IE10.

Reply:
Done